### DISCUSSION TITLE
NTRANS metafiles

### DISCUSSION THREAD

**Discussion by GwenChen-NOAA** [Unidata/gempak#148] [General]
Is NTRANS still included in gempak?  If it is no longer supported by gempak, how to view gempak metafiles created using `gdplot2_nc with device="nc | meta.file"`?  Any help is greatly appreciated.

**Comment by mzuranski** [comment]
I believe NTRANS is still included.  I see it on some of my installations, and while they aren't as current as this repo is now, searching the repo I see what appears to be the source code for it too.  If you already have Gempak installed, you should be able to verify NTRANS is included by running `which ntrans`.  For me that comes back as `~/NAWIPS/os/linux64/bin/ntrans`, right where it should be.

Note: I personally do not have any experience using NTRANS.  While it looks to be included still, I cannot make any statements to how well it works.

**Comment by GwenChen-NOAA** [comment]
Thanks for the info!  I don't have gempak installed on my machine.  I work
at NOAA/NCEP.  I'm working with NCO SDB <https://www.nco.ncep.noaa.gov/sdb/>
to get access to their installations.
